DB/src/main/java/io/deephaven/db/v2/sources/regioned/RegionedColumnSourceByte.java

```diff
@@ -78,14 +80,20 @@ public byte getByte(final long elementIndex) {
     static final class Partitioning extends RegionedColumnSourceByte<Values> {
 
         Partitioning() {
-            super(ColumnRegionByte.createNull(), Supplier::get);
+            super(ColumnRegionByte.createNull(), Supplier::get /* No need to interpose a deferred region in this case. */);
```

Comment:
I find comments inside statements annoying.   Could you consider splitting the argument to a separate line
and adding it as a // comment?

Reply:
Sigh, OK. I thought this was better/clearer.